tests/zst.rs

```diff
@@ -18,6 +18,6 @@ fn test_swap() {
 #[test]
 fn test() {
     let c = ArcArray::<(), _>::default((3, 4));
-    let mut d = c.clone();
+    let mut d = c;
```

Comment:
Cloning an Arc changes the ref count. It has to be assumed to be part of the test.

Reply:
I can revert the changes and then add some stuff to ignore lints on specific lines(I think)

Reply:
reverted and suppressed lints for function